src/rpc/rawtransaction.cpp

```diff
@@ -1133,7 +1133,7 @@ UniValue sendrawtransaction(const JSONRPCRequest& request)
     return hashTx.GetHex();
 }
 
-UniValue testmempoolaccept(const JSONRPCRequest& request)
+UniValue testmempoolaccept(const JSONRPCRequest& request) EXCLUSIVE_LOCKS_REQUIRED(mempool.cs)
```

Comment:
I fail to see how this could compile, since we never `LOCK(mempool.cs)` before calling into this, no?